.github/workflows/node_modules-publish.yml

```diff
@@ -0,0 +1,111 @@
+name: node_modules publish
+on:
+  workflow_run:
+    workflows: node_modules
+    types: [completed]
+
+jobs:
+  push:
+    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
```

Comment:
FTR, you don't need the `${{` wrapper in `if:`, that's always a condition.

Reply:
I think we talked about this in previous review rounds: I still prefer it because of some weird comments in the documentation that it's required for complex expressions.